### DISCUSSION TITLE
Allow customization for playfield border

### DISCUSSION THREAD

**Discussion by FrozenWaterrr** [ppy/osu#29509] [Ideas]
Right now, I prefer using a playfield border from a skin since the border is not as bright as the one that lazer provides in its settings, but I would like to use skin elements that don't feel like a "hack" since the skin playfield border essentially uses the HP bar to create the border effect. As someone who has difficulty seeing bright objects in front of a black screen, using the built-in playfield border is not very viable in gameplay since it becomes distracting when the hitcircles are near the edge of the playfield.

As a side note, is there a possibility of making the playfield border a skin element rather than a setting?

<table>
<tr>
<th>Aristia (Edit) </th>
<th> osu!lazer </th>
</tr>
<tr>
<td><img src="https://github.com/user-attachments/assets/353f2e17-0ff3-4b5e-b89e-326a6a2a7dd3"> </td>
<td> <img src="https://github.com/user-attachments/assets/e0a75536-85fe-410a-b906-72205f25b891"> </td>
</tr>
</table>

**Comment by peppy** [comment]
It should be converted to a skin element eventually, yes.
